### PR TITLE
steam: add page

### PR DESCRIPTION
**Pull request by thomas-m-d**
add page for valve's steam client

- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/steam.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# steam
+
+> Video game platform by Valve.
+> More information: <https://store.steampowered.com/>.
+
+- Launch Steam client in background:
+
+`steam &`
```
Comment (marchersimon) [inline pages/common/steam.md:8]:
```suggestion
```
Same as for the other page. It isn't really command specific. 

However I found <https://developer.valvesoftware.com/wiki/Command_Line_Options#Command-line_parameters_3>, which has a good amount of command line options for steam. It says it's for Windows, but I tested some and they all work for Linux too.

Comment (marchersimon) [reply pages/common/steam.md:8]: If you like to, go ahead. I'll convert the PR to a draft in between.

Comment (thomas-m-d) [reply pages/common/steam.md:8]: sounds good. hopefully i should have it done later today, but in the next couple days at the latest.

**Comment by thomas-m-d** [comment]
ok, i added what i considered the essential examples, and deleted the `steam &` example, so the page is at 5 examples total. there were a couple more i could add to get to 8, but i'm assuming the goal for tldr is to use as few examples as possible while still covering the essentials?

File: pages/common/steam.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# steam
+
+> Video game platform by Valve.
+> More information: <https://developer.valvesoftware.com/wiki/Command_Line_Options#Steam_.28Windows.29>.
+
+- Launch Steam client with logs (incl. errors) appearing in terminal:
```
Comment (marchersimon) [inline pages/common/steam.md:6]:
```suggestion
- Launch Steam and show debug messages in the terminal:
```

File: pages/common/steam.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`steam`
+
+- Launch Steam client with a debug console tab included in-app:
```
Comment (marchersimon) [inline pages/common/steam.md:10]:
```suggestion
- Launch Steam with a debug console tab included in-app:
```

File: pages/common/steam.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+`steam -console`
+
+- Log into Steam client with specified credentials:
```
Comment (marchersimon) [inline pages/common/steam.md:14]:
```suggestion
- Log into Steam with the specified credentials:
```

File: pages/common/steam.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+`steam -login {{username}} {{password}}`
+
+- Launch Steam in Big Picture Mode:
+
+`steam -tenfoot`
+
+- Shut down Steam client:
```
Comment (marchersimon) [inline pages/common/steam.md:22]:
```suggestion
- Exit Steam:
```

**Review by owenvoke — APPROVED** [review]
LGTM!

**Review by marchersimon — CHANGES_REQUESTED** [review]
> i'm assuming the goal for tldr is to use as few examples as possible while still covering the essentials?

You're absolutely right. Looks good to me so far, I just left a few suggestions below.

**Review by marchersimon — APPROVED** [review]

**Comment by thomas-m-d** [comment]
anything i need to do on my end to get this merged? looks like no?

**Comment by bl-ue** [comment]
No, you're good.  I'll take a look and then merge it 👍🏻

File: pages/common/steam.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
+# steam
+
+> Video game platform by Valve.
+> More information: <https://developer.valvesoftware.com/wiki/Command_Line_Options#Steam_.28Windows.29>.
+
+- Launch Steam and show debug messages in the terminal:
+
+`steam`
+
+- Launch Steam with a debug console tab included in-app:
```
Comment (bl-ue) [inline pages/common/steam.md:10]: This is still correct, is it not?
```suggestion
- Launch Steam and enable the debug console tab:
```

File: pages/common/steam.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+
+`steam -console`
+
+- Log into Steam with the specified credentials:
```
Comment (bl-ue) [inline pages/common/steam.md:14]: Can "Steam" here be replaced with the appropriate URL (e.g. https://steam.com)?  Unless that's not correct.

Comment (marchersimon) [reply pages/common/steam.md:14]: That would be <https://store.steampowered.com/>.

Comment (thomas-m-d) [reply pages/common/steam.md:14]: this command lets you skip the login prompt for the steam client gui, not something in your browser. normally you'd launch steam, enter your login info, then be able to access your games. this lets you skip the middle step.

File: pages/common/steam.md (hunk 6 above)
Comment (thomas-m-d) [reply pages/common/steam.md:10]: yes. however, i want to make it clear that the debug console is a tab in the steam app, to contrast it with the earlier example that gives you debug info in the bash shell you ran the command from. does something like:

`Launch Steam and enable its in-app debug console tab:`

look ok to you?

Comment (thomas-m-d) [reply pages/common/steam.md]: committed this and the previous change

File: pages/common/steam.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
+# steam
+
+> Video game platform by Valve.
+> More information: <https://developer.valvesoftware.com/wiki/Command_Line_Options#Steam_.28Windows.29>.
+
+- Launch Steam and show debug messages in the terminal:
```
[the remark this answers is not in the document]
Comment (thomas-m-d) [reply pages/common/steam.md]: committed this and the next change.

File: pages/common/steam.md (hunk 7 above)
Comment (bl-ue) [reply pages/common/steam.md:14]: Oh, I see.  Then let's leave it as-is. 👍🏻

**Comment by bl-ue** [comment]
Oh oops! I forgot about this one 😅

**Pull request merged**